Update backlog and appraisal tabs (#311)
artefactual/archivematica-docs: user-manual/appraisal/appraisal.rst

Preview file.

.. _analysis-objects:

Objects
^^^^^^^

The Objects tab of the Analysis pane displays format and size information about
selected files.

The Report view provides a list of information about the format, PRONOM unique
identifier (PUID), format group, number of files, and size of the selected
files.

.. figure:: images/analysis-objects-report.*
   :align: center
   :width: 80%
   :alt: The left-hand side of the image shows the backlog pane with a number of files selected. The right-hand side of the image shows information about those files, including the file format name, PUID, and size.

   Report showing information about the files that have been selected in the Backlog pane.

The Visualizations view provides two visualization options: the total number of
files for each format, or the total size of files for each format.

.. figure:: images/analysis-objects-visualizations.*
   :align: center
   :width: 80%
   :alt: The left-hand side of the image shows the backlog pane with a number of files selected. The right-hand side of the image shows a pie chart depicting the number of files for each format.

>    Visualization showing formats by total size of the selected files

Reviewer: ```suggestion
   Visualization showing total number of each format for the selected files
```
